### PR TITLE
Fixed an issue where tenant monitoring charts were displayed abnormally in cluster details

### PR DESCRIPTION
**Pull request by yang1666204**
## Summary

1. The tenant monitoring chart in the cluster details should only display the tenants currently existing under the cluster and the sys tenant.
image

2.Regenerate api.

### CODE REVIEW

File: ui/src/api/generated/base.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@
 import type { Configuration } from './configuration';
 // Some imports not used depending on template conditions
 // @ts-ignore
-import type { AxiosInstance, RawAxiosRequestConfig } from 'axios';
+import type { AxiosPromise, AxiosInstance, RawAxiosRequestConfig } from 'axios';
```
Comment (powerfooI) [inline ui/src/api/generated/base.ts:19]: This `AxiosPromise` import is unused in this file. It's OK to remove it.

**Review by powerfooI — CHANGES_REQUESTED** [review]

File: ui/src/api/generated/base.ts (hunk 1 above)
Comment (yang1666204) [reply ui/src/api/generated/base.ts:19]: In fact this line of code appears every time we regenerate, can we ignore it?

Comment (powerfooI) [reply ui/src/api/generated/base.ts:19]: OK, just ignore it if it's generated.

**Review by powerfooI — APPROVED** [review]
LGTM

**Pull request merged**